pkg/iterator/iter.go

```diff
@@ -242,20 +248,19 @@ func (iter *iterator) run(ctx context.Context) error {
 	return err
 }
 
-func (iter *iterator) getDefaultServiceAccount(ctx context.Context) (string, error) {
+func (iter *iterator) getDefaultServiceAccount(ctx context.Context) (string, string, error) {
```

Comment:
nit: Could you please make it clear in the function signature which return argument is the name and which is the namespace? e.g.
```
func (iter *iterator) getDefaultServiceAccount(ctx context.Context) (namespace string, name string, err error)
```